src/state/opamFileTools.ml

```diff
@@ -524,8 +524,8 @@ let t_lint ?check_extra_files ?(check_upstream=false) ?(all=false) t =
        (t.install <> [] || t.remove <> [] || t.url <> None ||
         t.extra_sources <> []));
     cond 47 `Warning
-      "Synopsis (or description first line) should start with a capital and \
-       not end with a dot"
+      "Synopsis (or 'descr' section first line) should start with a capital \
```

Comment:
```suggestion
      "Synopsis should start with a capital \
```
Should we maybe entierly omit this, given this is opam 1.2 stuff?

Reply:
It would make the warning message clearer and more to the point

Reply:
Does all packages have a synopsis ? Lint 57 just checks if synopsis and descr are not both empty

Reply:
Good question. I was under the impression that `synopsis` was "mandatory" (would raise a warning if missing), but now that https://github.com/ocaml/opam/pull/5070 has been merged it’s not the case anymore.

I think it should be mandatory given currently if it is not present, `opam list` will not show the "first line of the description field" as this W47 might indicate:
```
opam list hll.3.16
# Packages matching: name-match(hll) & version-match(3.16) & (installed | available)
# Package # Installed # Synopsis
hll.3.16  --
```
(`hll.3.16` currently does not have `synopsis` but a one line description)

Reply:
#5070 doesn't change the lint behaviour, it only fixes double linting trigger (empty synopsis | description that trigger no capital one).

Cited list behaviour for me is a list issue as we didn't defined that synopsis is mandatory & non empty (which is the case of hll)